### PR TITLE
Disable ee for version-2.25

### PR DESCRIPTION
**Pull request by markusdregi**
**Starting point**
This PR reverts commit 8c41328ed8c8b3a10a3a31b665bc6f46dca80abc because ports are yet to be opened on-premise. This was done by cherry-picking the revert commit from the `version-2.24` branch and fixed some conflicting imports.

**Status**
However, some of the unit tests where only ran for the default evaluator. Which caused failing tests when reverting to using the legacy evaluator. This needs to be fixed...

### CODE REVIEW

File: tests/status/test_legacy_tracker.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def check_expression(original, path_expression, expected, msg_start):
             "poly_example",
             [
                 ENSEMBLE_SMOOTHER_MODE,
-                "--disable-ensemble-evaluator",
+                "--enable-ensemble-evaluator",
```
Comment (markusdregi) [inline tests/status/test_legacy_tracker.py]: I have yet to figure out why, but it seems like this test is leaking into the `failing_poly_example`. If I remove all the other test cases but `failing_poly_example` it passes. If I reintroduce all cases but this one it passes (but the top one still fails). If I reintroduce this case as well, `failing_poly_example` starts failing. If I revert this case also to run without the ensemble evaluator, `failing_poly_example` starts passing again.

It seems like mixing tests using and not using the ensemble evaluator does not end well 🤔

Comment (jondequinor) [reply tests/status/test_legacy_tracker.py]: That was due to the `FeatureFlag` not being reset.

Comment (jondequinor) [reply tests/status/test_legacy_tracker.py]: Such is the life with globals.

Comment (markusdregi) [reply tests/status/test_legacy_tracker.py]: I'm dropping the commit then 👍🏻

**Review by markusdregi — COMMENTED** [review]
These changes makes sense 👍🏻

**Review by jondequinor — APPROVED** [review]
LGTM

**Pull request merged**